app/main.ipynb

```diff
@@ -151,17 +199,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "days_of_week = [\"Monday\", \"Tuesday\", \"Wednesday\", \"Thursday\", \"Friday\", \"Saturday\", \"Sunday\"]\n",
+    "order_data[\"Order Day Of The Week\"] = order_data[\"datetime\"].dt.dayofweek\n",
+    "order_data[\"Order Day Of The Week\"] = order_data[\"Order Day Of The Week\"].apply(lambda x: days_of_week[x])\n",
+    "\n",
+    "order_data[\"Order Day Of The Week\"] = pd.Categorical(order_data[\"Order Day Of The Week\"], categories=days_of_week)\n",
+    "\n",
+    "order_data.groupby(\"Order Day Of The Week\", observed=False)[\"Item Price\"].sum().plot.bar(xlabel=\"Order Day Of The Week\", ylabel=\"Item Prices\", title=\"Prices At Days Of The Week\", color=\"purple\", width=0.9)\n",
```

Comment:
Ensure that the `categories` parameter in `pd.Categorical` matches the expected order of days to avoid any misalignment in the bar chart.